docs/Utilities.md

```diff
@@ -201,10 +201,16 @@ Memoizing functions allow for quicker and computationally cheaper calculations w
 
 ### `get_contract_class`
 
-A helper method that returns the contract class from the given path. To capture the contract class, simply add the contract path as an argument like this:
+A helper method that returns the contract class from the contract's name. To capture the contract class, simply add the contract's name as an argument like this:
 
 ```python
-contract_class = get_contract_class('path/to/contract.cairo')
+contract_class = get_contract_class('ContractName')
+```
+
+If multiple contracts exist with the same name, then the contract's path must be used instead of the name. To pass the contract's path:
```

Comment:
```suggestion
If multiple contracts exist with the same name, then the contract's path must be passed along with the `is_path` flag instead of the name. To pass the contract's path:
```

We should be explicit so users don't miss the flag